tests/test_product_stability.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import print_function
+
+import argparse
+import glob
+import os.path
+import sys
+
+import ssg.products
+
+import tests.test_profile_stability as stability
```

Comment:
Instead of importing the test_profile_stability I would prefer to extract the shared code to a separate file.